### PR TITLE
Fix language detection runtime error

### PR DESCRIPTION
**Pull request by urbantrout**
I get a runtime exception when trying to detect language of a mp3 file:

```
RuntimeError: Given groups=1, weight of size [1280, 128, 3], expected input[1, 80, 3000] to have 128 channels, but got 80 channels instead
```
This change fixes that.

See: https://github.com/openai/whisper/pull/1764#issuecomment-1817497095

### CODE REVIEW

**Comment by urbantrout** [comment]
Side note: This happened with the `turbo` model.

File: app/openai_whisper/core.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def language_detection(audio):
     audio = whisper.pad_or_trim(audio)
 
     # make log-Mel spectrogram and move to the same device as the model
-    mel = whisper.log_mel_spectrogram(audio).to(model.device)
+    mel = whisper.log_mel_spectrogram(audio, n_mels=128).to(model.device)
```
Comment (nickcaballero) [inline app/openai_whisper/core.py:51]: The value depends on the model. See https://github.com/openai/whisper/discussions/1778#discussioncomment-11008382. Use `model.dims.n_mels` instead.

Comment (urbantrout) [reply app/openai_whisper/core.py]: Thanks! I updated the PR

**Comment by ahmetoner** [comment]
#259

**Pull request closed without merging**